sql/sql_expression.h

```diff
@@ -504,23 +504,31 @@ struct GetAllColumnsOutput {
 
 struct ColumnFilter {
     typedef std::function<ColumnName (const ColumnName &)> Exec;
+    bool init;
    
-    ColumnFilter() 
+    ColumnFilter() : init(false)
     {
-
     }    
 
-    ColumnFilter(Exec exec)
-        : exec(std::move(exec))
+    ColumnFilter(Exec exec_)
+        : exec(std::move(exec_))
     {
+        init = true;
+    }
+
+    static ColumnFilter identity() {
```

Comment:
I was thinking of something like that:
.h
`static ColumnFilter identity`
.cc
```
ColumnFilter::identity = ColumnFilter(
[](const ColumnName & columnName) {
   return columnName;
});
```
Everywhere else
`ColumnFilter::identity`
And I would have removed the unnecessary default constructor and the init member.